cmd/main.go

```diff
@@ -69,11 +70,14 @@ func main() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	flag.DurationVar(&requeueTime, "requeue-time", RequeueDuration,
-		"Duration for the validation reconciliation loop. "+
+		"The maximum times it takes between reconciliations of DNS Record "+
```

Comment:
requeue-time feels slightly off here, this should be called something like "maximum-requeue-time" for more clarity.

Reply:
renamed